ports/matio-cpp/vcpkg.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "matio-cpp",
```

```suggestion
  "name": "libmatio-cpp",
```
There is already a repository named `matio-cpp` on https://repology.org/projects/, which points to [https://github.com/fd00/yacp](https://github.com/fd00/yacp). Additionally, the repository named `libmatio-cpp` on https://repology.org/projects/ points to the homepage URL mentioned in the current PR.

---

Indeed I checked repology.org for reference, but I'm fine changing it to `libmatio-cpp`